README.md

```diff
@@ -197,7 +197,22 @@ sortable('.sortable', {
   maxItems: 3 // Defaults to 0 (no limit)
 });
 ```
+### acceptFrom
```

Comment:
Somehow your rebase did not work, this should not be in your PR `acceptFrom` and I changed and moved it as well (see the current readme). Can you please try to rebase again (and pull master first. 😉in case you forgot). Thanks.